src/databricks/labs/ucx/assessment/queries/failure_summary.sql

```diff
@@ -0,0 +1,131 @@
+-- viz type=table, name=Failure Summary, columns=issue,component,issue_count,issue_percentage_jobs,issue_percentage_clusters,issue_percentage_gis,issue_percentage_pipelines
+-- widget title = Failure Summary, col=4, row=0, size_x=5, size_y=3
+WITH failuretab (failures, component) AS (
```

Comment:
can you create this rather as a view, which is added during the assessment step? we'll be using this view in the other downstream reports

Reply:
@nfx - can you please show me some examples of create view in the existing code which i can refer to?